client/state/billing-transactions/individual-transactions/test/actions.js

```diff
@@ -0,0 +1,82 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { requestBillingTransaction, clearBillingTransactionError } from '../actions';
+import {
+	BILLING_TRANSACTION_REQUEST,
+	BILLING_TRANSACTION_REQUEST_FAILURE,
+	BILLING_TRANSACTION_REQUEST_SUCCESS,
+} from 'state/action-types';
+import useNock from 'test/helpers/use-nock';
+
+describe( 'actions', () => {
+	describe( '#requestBillingTransaction', () => {
+		const transactionId = 12345678;
+
+		describe( 'success', () => {
+			useNock( nock => {
+				nock( 'https://public-api.wordpress.com:443' )
+					.persist()
+					.get( '/rest/v1.1/me/billing-history/receipt/' + transactionId + '?format=display' )
+					.reply( 200, { data: 'receipt' } );
```

Comment:
How about specifying a representation of a real receipt here instead of the `receipt` string? I find those helpful when reading the code afterwards.